client/app/components/packages/landuse-form/edit.js

```diff
@@ -100,6 +102,24 @@ export default class LandUseFormComponent extends Component {
     relatedAction.deleteRecord();
   }
 
+  @action
+  addSitedatahForm(changeset) {
+    const newSitedatahForm = this.store.createRecord('sitedatah-form', {
+      landuseForm: this.landuseForm,
+    });
+
+    addToHasMany(changeset, 'sitedatahForms', newSitedatahForm);
+  }
+
+  @action
+  removeSitedatahForm(sitedatahForm, changeset) {
+    removeFromHasMany(changeset, 'sitedatahForms', sitedatahForm);
+
+    this.recordsToDelete.push(sitedatahForm);
```

Comment:
This feels wrong. It seems like it's done to trigger an observer somewhere. Not a big deal because we probably do it elsewhere but should find a better way soon.

Reply:
I agree. One reason for this design currently is to "queue" up the records for actual deletion in the backend. 
However I think it may also be contributing to a current bug with the state of the save button (the save button isn't being appropriately disabled when sitedatahForm are removed)